docs/command-center.md

```diff
@@ -0,0 +1,33 @@
+# Command Center
+
+Opsdroid Command Center is the dashboard that allows you to interact with Opsdroid without the need to have any connector installed (other than the default websocket one). The dashboard allows you to enable/disable modules as well see your active modules and installed configuration from within your browser.
+
+Opsdroid Command Center is powered by the [opsdroid web](https://github.com/opsdroid/opsdroid-web) package, you need to first install the package using `npm`.
```

Comment:
We should handle this so that the web interface gets installed via Python. Maybe as part of our build process here or adding some packaging to PyPI in opsdroid web.

Reply:
I removed this docs for now since I need to fix opsdroid web first, I tried running it and it seems its very broken on the latest node version